pkg/apis/numaflow/v1alpha1/const.go

```diff
@@ -147,7 +147,7 @@ const (
 	DefaultKeyForNonKeyedData = "NON_KEYED_STREAM"
 
 	// Default gRPC max message size
-	DefaultGRPCMaxMessageSize = 20 * 1024 * 1024
+	DefaultGRPCMaxMessageSize = 4 * 1024 * 1024
```

Comment:
Is there any impact (e.g. performance) if we keep it 20?

Reply:
there shouldn't be any performance impact. i think he wanted to lower the value because we will hardly cross 4MB since it is streaming for both Reduce and UDSink.

@whynowy do you want 20MB as default?

Reply:
If there's no performance impact, keeping a higher value makes less trouble.

Reply:
does the jetstream support messages as big as 20MB?

Reply:
JetStream defaults max msg size to 1 MB, it's configurable to make it larger. If setting the value larger here does not make any negative impact, we should do it.

Reply:
@yhl25, let's default to 20MB.

Reply:
Okayy